chapters/synchronous.tex

```diff
@@ -223,22 +223,23 @@ \subsection{Argument Restrictions (Component Expression)}\label{argument-restric
 \begin{definition}[Component expression]\label{def:component-expression}\index{component!expression (argument restriction)}
 A component expression is a \lstinline[language=grammar]!component-reference! which is a valid expression, i.e., not referring to models or blocks with equations.
 % "an element of records" below looks strange:
-In detail, it is an instance of a (a) base type, (b) derived type, (c) record, (d) an array of such an instance (a-c), (e) one or more elements of such an array (d) defined by index expressions which are parameter expressions (see below), or (f) an element of records.
+In detail, it is an instance of a (a) base type, (b) derived type, (c) record, (d) an array of such an instance (a-c), (e) one or more elements of such an array (d) defined by index expressions which are evaluable expressions (see below), or (f) an element of records.
 \begin{nonnormative}
 The essential features are that one or several values are associated with the instance, that start values can be defined on these values, and that no equations are associated with the instance.
 A component expression can be constant or can vary with time.
 \end{nonnormative}
 \end{definition}
 
 In the following sections, when defining an operator with function calling syntax, there are some common restrictions being used for the input arguments (operands).
-For example, an input argument to the operator may be required to be a component expression (\cref{def:component-expression}) or parameter expression (\cref{variability-of-expressions}).
+For example, an input argument to the operator may be required to be a component expression (\cref{def:component-expression}) or evaluable expression (\cref{variability-of-expressions}).
 To emphasize that there are no such restrictions, an input argument may be said to be just an \emph{expression}.
 
 \begin{nonnormative}
 The reason for restricting an input argument to be a component expression is that the start value of the input argument is returned before the first tick of the clock of the input argument and this
 is not possible for a general expression.
 
-The reason for restricting an input argument to be a parameter expression is that the value of the input argument needs to be evaluated during translation, in order that clock analysis can be performed during translation.
+The reason for restricting an input argument to be a evaluable expression is that the value of the input argument needs to be evaluated during translation, in order that clock analysis can be performed during translation.
+In some cases there is a special handling of parameter expressions, which indicates that their values are not needed during translation.
```

Comment:
Maybe better?
```suggestion
In some cases there is special handling of parameter expressions, indicating that their values are not needed during translation.
```

Reply:
Will reformulate completely.

Reply:
Better now?

Reply:
I'll start over with suggestions for the new formulation instead.  Resolving.